lib/engine/ownable.rb

```diff
@@ -10,8 +10,19 @@ def owned_by?(entity)
       owner == entity || owner&.owner == entity || owner == entity&.owner
     end
 
+    # avoid infinite recursion for 1841
     def player
-      owner&.player? ? owner : owner&.player
+      owner_chain = [owner]
```

Comment:
what is this? i can't really understand it

Reply:
maybe call it 

owners = {owner: true}

Reply:
OK, although maybe:

chain = {owner: true}

since a "chain of ownership" is a concept the 1841 rules specifically mention.